python/sglang/srt/layers/quantization/awq.py

```diff
@@ -58,8 +62,13 @@
     )
 
     warnings.warn(f"HIP does not support fused_marlin_moe currently.")
+elif _is_cpu and _is_cpu_amx_available:
+    from sglang.srt.layers.amx_utils import (
+        SGLANG_USE_CPU_INT4_W4A8,
+        _amx_process_int4_packed_qweight_after_loading,
```

Comment:
use `_amx_process_weight_after_loading` the same API for the same purpose.

Reply:
sure, have refined here with `_amx_process_weight_after_loading`